runtime/js/99_main.js

```diff
@@ -1029,6 +1031,7 @@ function bootstrapWorkerRuntime(
     closeOnIdle = runtimeOptions[14];
 
     performance.setTimeOrigin();
+    event.setEventTargetData(performance);
```

Comment:
Also I don't understand this bit - does it mean that we need to call `setEventTargetData` for every single thing that `extends` `Event`?